### PR TITLE
Better handle error

### PR DESCRIPTION
**Pull request by p16i**
Currently, if something goes wrong in files such as `youpin.js`, the server doesn't stop which is difficult to what is actually happening when testing with Facebook Messager.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -58,6 +58,11 @@ var youpin;
 new api_lib(API_URI, API_USERNAME, API_PASSWORD).then(function(api) {
   // Youpin bot
   youpin = require('./youpin.js')(m, api, conversation, API_USER_ID);
+}).catch(err => {
+  console.log(err)
+  if(process.env.NODE_ENV != "production"){
```
Comment (supasate) [inline app.js]: Use `!==`.
Add spaces before `(` and after `)`

**Review by supasate — CHANGES_REQUESTED** [review]

File: app.js [hunk 2]
```diff
@@ -106,5 +111,5 @@ app.post('/webhook/', function(req, res) {
 
 
 app.listen(app.get('port'), function() {
-  console.log(`Node app is running on port ${app.get('port')}`);
+    console.log(`Node app is running on port ${app.get('port')}`);
```
Comment (supasate) [inline app.js]: 2 spaces are alright. No need to change.

**Review by supasate — CHANGES_REQUESTED** [review]

File: app.js [hunk 3]
```diff
@@ -60,7 +60,7 @@ new api_lib(API_URI, API_USERNAME, API_PASSWORD).then(function(api) {
   youpin = require('./youpin.js')(m, api, conversation, API_USER_ID);
 }).catch(err => {
   console.log(err)
-  if(process.env.NODE_ENV != "production"){
+  if( process.env.NODE_ENV !== "production" ){
     process.exit(1);
```
Comment (supasate) [inline app.js:64]: I mean `if (process.env.NODE_ENV !== "production") {`.

**Review by supasate — CHANGES_REQUESTED** [review]

File: app.js (hunk 3 above)
Comment (p16i) [reply app.js:64]: 😢

File: app.js (hunk 2 above)
Comment (supasate) [reply app.js]: @heytitle this line is not fixed yet

**Review by supasate — CHANGES_REQUESTED** [review]

**Comment by tvirot** [comment]
LOL, @supasate is becoming a lint bot 😆

**Review by supasate — APPROVED** [review]

**Pull request merged**